extension_templates/classification.py

```diff
@@ -252,8 +252,9 @@ def get_test_params(cls, parameter_set="default"):
         -------
         params : dict or list of dict, default = {}
             Parameters to create testing instances of the class
-            Each dict are parameters to construct an "interesting" test instance, i.e.,
-            `MyClass(**params)` or `MyClass(**params[i])` creates a valid test instance.
+            Each dict are parameters to construct an "interesting"
+            and viable test instance, i.e., `MyClass(**params)` or
```

Comment:
what do you mean by "viable"?

Also, if you change it here, we need to change it in all other instances of this docstring.

Reply:
By viable I meant the set should also be feasible, certain interesting cases might require more resources than, let's say, the default set. This set should be avoided since they are not viable.

But this distinction is made again in the second half, so I'll revert back to what it originally was.